### DISCUSSION TITLE
Ability to drag images to other applications

### DISCUSSION THREAD

**Discussion by cibernicola** [zanllp/infinite-image-browsing#572] [Ideas]
I don't know if it is possible or should work already, but it would be great to be able to drag images from IIB to other windows (browsers), to speed up the workflow. 
Currently, at least in my case, when dragging an image from IIB to, for example, a browser window running WebUI (Forge), nothing happens.
Note: From the browser it is possible.

**Comment by zanllp** [comment]
I will try to see if it is feasible.
